resources/letters/attributes/local_file_path.yml

```diff
@@ -0,0 +1,6 @@
+type: string
+
+description: >-
+  A link to a remote file to be used for the postcard template.
```

Comment:
Lets put `local_file_path.yml` in `shared/attributes` and add a comment block that shows how to use it, with `allOf` and the `pattern: "ridiculouslycomplexregex"`.

The description can be `A link to a remote file to be used for the template.`

The goal is to highlight the similarities and differences between form factors....

Reply:
Ah, so the reason there are two `local_file_path` files is that they are slightly different between letter & postcard (letter does not allow PNG or JPEG uploads).